src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int NAME_LENGTH = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        TreeSet<Character> stringSet = new TreeSet<>();
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            int inputChar = 0;
+            while ((inputChar = bufferedReader.read()) != -1) {
+                if (Character.isAlphabetic(inputChar)) {
```

Comment:
```suggestion
                if (Character.isLetter(inputChar)) {
```
read about the difference)